portal/templates/redesign/about_new.html

```diff
@@ -0,0 +1,138 @@
+            <div class="col-sm-6">
+                <h3>Want to get involved?</h3><br>
+                <p class="body-text">We are open source, so anyone can ask to contribute. You can play with game-running JavaScript, Python/Django, animation
+                    using SGV and Raphael, and a lot more. We&rsquo;d like input from all sorts of backgrounds, whether you&rsquo;re: a programmer looking for a
+                    creative outlet; a teacher hoping to shape the resources;
+                    <a target="_blank" href="https://crowdin.com/project/code-for-life/invite/public?show_welcome&d=876e8507d82d9e07a09b4696a88c0c02479434">
+                        a polyglot who&rsquo;d like to help with translation</a>;
+                    or even a pupil putting your skills to the test.</p><br><br><br>
+                <h4>Developers</h4><br>
+                <p class="body-text">To contribute, head over to <a target="_blank" href="https://github.com/ocadotechnology/codeforlife-portal">GitHub</a>,
+                    check out the issue tracker, and get started. There you can suggest new features or assign yourself an issue to develop (you&rsquo;ll find
+                    more info about how to do this on GitHub).</p><br><br><br>
+                <h4>Teachers, parents, and creatives</h4><br>
+                <p class="body-text">Please get in touch through our <a href="{% url 'help_new' %}">contact</a> form and let us know how you would like to get involved.</p>
+            </div>
+        </div>
+    </div>
+    <section class="col-sm-7 col-center">
+        <h3>We would like to thank our friends who <br>have contributed to this initiative</h3>
+        <div class="row row--regular image-row">
+            <img src='{% static "portal/img_new/10x_logo.png" %}'>
+            <img src='{% static "portal/img_new/bcs_logo.png" %}'>
+            <img src='{% static "portal/img_new/icl_logo.png" %}'>
+            <img src='{% static "portal/img_new/barefoot_logo.png" %}'>
+            <img src='{% static "portal/img_new/mc_saatchi_logo.png" %}'>
+            <img src='{% static "portal/img_new/hope_logo.png" %}'>
+            <img src='{% static "portal/img_new/gla_logo.png" %}'>
+            <img src='{% static "portal/img_new/pressure_cooker_logo.png" %}'>
+        </div>
+        <p class="small-text">10X, BCS Academy of Computing, Barefoot Computing, Computing at School, The National Museum of Computing, Imperial College London,
+            M&C Saatchi, Alvaro Ramirez, Jason Fingland, Ramneet Loyall, Sharon Harrison, Keith Avery, Dale Coan, Rob Whitehouse, Mandy Nash, Tanya
+            Nothard, Matt Trevor, Moy El-Bushra, Richard Siwiak, Peter Tondrow, Liz Pratt, Pressure Cooker Studios, GAL Education, Hope Education.</p>
+    </section>
+</div>
+
+<section class="background col-sm-5 col-center">
+    <h3>Dedicated to Sharon Harrison</h3>
+    <img class="img-circle" src='{% static "portal/img_new/sharon_harrison.jpg" %}'>
```

Comment:
img-circle is an other unsemantic classname.  Why do you want a circular image here?